src/main/kotlin/Main.kt

```diff
@@ -1,3 +1,12 @@
+val ListOfArchives = mutableListOf<Archive>()
 fun main(args: Array<String>) {
-    println("Hello World!")
-}
\ No newline at end of file
+
+    println("\n --------------------- ")
+    println("|| Программа ЗАМЕТКИ ||")
+    println(" --------------------- ")
+    
+    for (a in ListOfArchives)
+        println("\t${ListOfArchives.indexOf(a)}. ${a.name}")
```

Comment:
🍏 Еще можно использовать метод forEachIndexed:
```suggestion
    ListOfArchives.forEachIndexed { index, archive ->
        println("\t${index}. ${archive.name}")
    }
```